triedb/pathdb/difflayer.go

```diff
@@ -113,7 +113,7 @@ func (dl *diffLayer) node(owner common.Hash, path []byte, hash common.Hash, dept
 			// bubble up an error here. It shouldn't happen at all.
 			if n.Hash != hash {
 				dirtyFalseMeter.Mark(1)
-				log.Error("Unexpected trie node in diff layer", "owner", owner, "path", path, "expect", hash, "got", n.Hash)
+				log.Warn("Unexpected trie node in diff layer", "owner", owner, "path", path, "expect", hash, "got", n.Hash)
```

Comment:
Hi, there are other related error level logs, could you help to downgrade them?
1. "Unexpected trie node in diff layer"
2. "Unexpected trie node in clean cache"
3. "Unexpected trie node in disk"
4. "Unexpected trie node in async node buffer"